### ISSUE TITLE
Show only The Movie not all seasons

### ISSUE THREAD

**Issue by dertuerke** [slimndap/wp-theatre#284]
Hello, how can I display only a list of movies without the game times?
Now, if I have a movie with multiple times, it will be shown 10 times. But I want the movie to be shown only once.

Thanks for your help

**Comment by slimndap** [comment]
Hi!

You can setup a page with the following shortcode on it:

[wpt_productions end_after=now]

This will show a list of ‘movies’ instead of ‘showtimes’.

Jeroen

**Comment by dertuerke** [comment]
Hi @slimndap, the code dosnt work :S
`[wpt_events_grid end_after="now"]`

**Comment by slimndap** [comment]
I see. The syntax for the [events grid extension](https://wp.theater/downloads/cascading-events-for-theater/) is a bit different:

`[wpt_events_grid mode="compact" end_after="now"]`

**Comment by dertuerke** [comment]
Hi where you found the "mode" i dont see it????

**Comment by slimndap** [comment]
I am afraid it is undocumented. It is not part of the Theater for WordPress
plugin. It is also the only undocumented feature of the events grid
extension.

**Comment by dertuerke** [comment]
I was just wondering where you found this. I found the same thing, only directly in the code.

That's why I wanted to ask again where you found it :)

Thx for your help @slimndap
